### DISCUSSION TITLE
How to add Google Analytics in the theme?

### DISCUSSION THREAD

**Discussion by shikharvaish28** [LekoArts/gatsby-themes#875] [Questions] [answered]
I am using the minimal blog theme and want to add Google Analytics to the webpage. Can anyone please help me with how can I go about it?

**Comment by LekoArts** [comment, marked as answer]
Install https://www.gatsbyjs.com/plugins/gatsby-plugin-google-gtag/ and add it to your `gatsby-config.js`
